app/controllers/dashboard_controller.rb

```diff
@@ -20,26 +20,6 @@ def index
     redirect_to(:action => 'show')
   end
 
-  def dialog_definition
-    name = params[:name].gsub(/[^a-z_]/, '')
-    definition = load_dialog_definition(name, params[:class])
-    if definition.present?
-      render :json => { :dialog => definition }
-    else
-      head :not_found
-    end
-  end
-
-  def load_dialog_definition(name, klass)
-    plugin = Vmdb::Plugins.find { |plug| plug.name.chomp('::Engine') == klass }
-    if plugin.present?
-      name = plugin.root.join('dialogs', "#{name}.json")
```

Comment:
This is the key line for me, because there aren't any plugins with a `dialogs` directory at the root.